### DISCUSSION TITLE
Exclude media quickly that will not be part of the meeting media

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#1002] [Ideas] [answered]
Session: Do your best in ministry. When quoting the lff brochure, the media is downloaded. 
Suggestion, put an X on the media to delete it from the preview pane quickly.

![image](https://user-images.githubusercontent.com/90984283/210286757-8bfb7f0c-6772-43ff-97a5-ef14399a5b94.png)

**Comment by mtdvlpr** [comment, marked as answer]
@e-Feitosa, that's my fault. When adding the option to exclude media from the book, I removed the logic to exclude media from the brochure. That will be fixed in the new release. As for your suggestion. What would the use case be? It's usually 1 or 2 images that you can easily scroll past.

**Reply by e-Feitosa** [reply]
The suggestion of the X in the image was to exclude the media from the session do your best in ministry that were downloading too much.

**Reply by mtdvlpr** [reply]
I think we'll release this fix soon, given the same thing will occur next week. For now, if you are using a congregation server you could hide the media items beforehand and then they wouldn't be downloaded. I think with this fix, the suggestion is no longer valid, am I right?

**Reply by e-Feitosa** [reply]
With the update, it will resolve this issue 100%. Thank you very much for your work and for the attention always so fast
